kibana/fleet.go

```diff
@@ -29,18 +29,19 @@ import (
 )
 
 const (
+	fleetAgentAPI             = "/api/fleet/agents/%s"
 	fleetAgentPoliciesAPI     = "/api/fleet/agent_policies"
 	fleetAgentPolicyAPI       = "/api/fleet/agent_policies/%s"
+	fleetAgentsAPI            = "/api/fleet/agents"
 	fleetAgentsDeleteAPI      = "/api/fleet/agent_policies/delete"
 	fleetEnrollmentAPIKeysAPI = "/api/fleet/enrollment_api_keys" //nolint:gosec // no API key being leaked here
-	fleetAgentsAPI            = "/api/fleet/agents"
-	fleetAgentAPI             = "/api/fleet/agents/%s"
-	fleetUnEnrollAgentAPI     = "/api/fleet/agents/%s/unenroll"
-	fleetUpgradeAgentAPI      = "/api/fleet/agents/%s/upgrade"
-	fleetFleetServerHostsAPI  = "/api/fleet/fleet_server_hosts"
 	fleetFleetServerHostAPI   = "/api/fleet/fleet_server_hosts/%s"
+	fleetFleetServerHostsAPI  = "/api/fleet/fleet_server_hosts"
 	fleetPackagePoliciesAPI   = "/api/fleet/package_policies"
+	fleetUnEnrollAgentAPI     = "/api/fleet/agents/%s/unenroll"
 	fleetUninstallTokensAPI   = "/api/fleet/uninstall_tokens" //nolint:gosec // NOT the "Potential hardcoded credentials"
+	fleetUpgradeAgentAPI      = "/api/fleet/agents/%s/upgrade"
+	fleetAgentDownloadSources = "/api/fleet/agent_download_sources"
```

Comment:
For consistency, we should add an `API` suffix.
```suggestion
	fleetAgentDownloadSourcesAPI = "/api/fleet/agent_download_sources"
```